README.md

```diff
@@ -49,6 +49,28 @@ Now you can run the test suite by typing the following:
 	cd client
 	node mbTests.js
 
+### Using Microbit Firmata as an NPM library
+
+Create a new package.json if you don't have one already by running `npm init` and following the prompts, then install this library
+as a dependency with:
+
+```
+npm install --save https://github.com/microbit-foundation/microbit-firmata.git
+```
```

Comment:
I assume by default this will grab the latest version from the main branch?

If that's the case, we should encourage to use a tag instead, as we cannot ensure we are not pushing breaking changes to the main branch and should use something like semver to flag this.

Reply:
Agreed, we can signal this in the README perhaps, as _technically_ checking out master is a valid thing to do, just not recommended.

We should be able to specify tags via `#` delimiting it on the URL:
`https://github.com/microbit-foundation/microbit-firmata.git#v1.3.0`

Not ideal for npm packages, as we should properly publish on each tag really, but as a workaround for not having any current npm presence this should be fine.

Reply:
We'll also have to create a v1.3.1 with the required package.json so it will actually work, too.